docker/Dockerfile

```diff
@@ -127,7 +127,7 @@ RUN cd ${MINIFI_BASE_DIR} \
     -DDISABLE_CURL=${DISABLE_CURL} -DDISABLE_JEMALLOC=${DISABLE_JEMALLOC} -DDISABLE_CIVET=${DISABLE_CIVET} \
     -DDISABLE_EXPRESSION_LANGUAGE=${DISABLE_EXPRESSION_LANGUAGE} -DDISABLE_ROCKSDB=${DISABLE_ROCKSDB} \
     -DDISABLE_LIBARCHIVE=${DISABLE_LIBARCHIVE} -DDISABLE_LZMA=${DISABLE_LZMA} -DDISABLE_BZIP2=${DISABLE_BZIP2} \
-    -DDISABLE_SCRIPTING=${DISABLE_SCRIPTING} -DDISABLE_CONTROLLER=${DISABLE_CONTROLLER} .. \
```

Comment:
I would go for `RelWithDebInfo` instead

Reply:
The problem with `RelWithDebInfo` was that it resulted in way too big docker images, due to the large minifi and minificontroller libraries when debuginfo was included. With Release the final image is 232MB with AWS and Kafka included, but with debuginfo the image size becomes more thatn 600MBs.